test/cli/run/if-present.test.ts

```diff
@@ -0,0 +1,26 @@
+import { it, expect } from "bun:test";
+import { bunEnv, bunExe } from "harness";
+
+it("should not error using `bun --if-present`", () => {
```

Comment:
this needs the positive test too:
- `bun --if-present works`
- `bun run --if-present works`
- `bun --if-present ./works.ts`
- `bun run --if-present ./works.ts`